webapp/components/sidebar_header_dropdown.jsx

```diff
@@ -41,6 +41,7 @@ export default class SidebarHeaderDropdown extends React.Component {
         this.handleAboutModal = this.handleAboutModal.bind(this);
         this.aboutModalDismissed = this.aboutModalDismissed.bind(this);
         this.toggleAccountSettingsModal = this.toggleAccountSettingsModal.bind(this);
+        this.switchTeam = this.switchTeam.bind(this);
```

Comment:
they are removed in the Team Sidebar PR :)

but hey I can just rebase and remove this later on if this makes it to master before team sidebar